docker/compose-tracing-v1.yml

```diff
@@ -63,7 +63,7 @@ services:
       file: compose-spec-l2-services.yml
       service: coordinator
     environment:
-      config__override__l2-network-gas-pricing__json-rpc-pricing-propagation__disabled: true
+      config__override__l2-network-gas-pricing__json-rpc-pricing-propagation__disabled: false
```

Comment:
@jpnovais Why do we disable this? Do u think its okay to re-enable it? If okay, maybe we could just remove this line